pyproject.toml

```diff
@@ -3,19 +3,19 @@ requires = ["setuptools>=61.0.0", "wheel"]
 
 [project]
 name = "cbcbeatx"
-version = "0.7.0"
+version = "0.7.0.dev0"
```

Comment:
Why dev?

Reply:
To Give it a unique version different than the one already released (0.7.0)